source/brailleDisplayDrivers/seikantk.py

```diff
@@ -212,39 +210,37 @@ def _processCommand(self, command: bytes, arg: bytes) -> None:
 			log.warning(f"Seika device has received an unknown command {command}")
 
 	def _handInfo(self, arg: bytes):
-		self.numCells = arg[2]
-		self.numBtns = arg[1]
+		self.numBtns = arg[0]
```

Comment:
It would be handy to mention which part of the spec this comes from, and the same comment for the other handler functions.